Application/EdFi.Ods.Api.IntegrationTestHarness/appsettings.json

```diff
@@ -0,0 +1,74 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning",
+      "Microsoft": "Warning"
+    }
+  },
+  "Urls": "http://localhost:8765",
+  "AllowedHosts": "*",
+  "DefaultApplicationName": "Default Sandbox Application",
```

Comment:
We should avoid putting so much on the root, and group this stuff into sections for easier management.